recipes-core/images/aos-image-initramfs.bbappend

```diff
@@ -0,0 +1,18 @@
+AOS_INITRAMFS_SCRIPTS += " \
+    initramfs-module-lvm \
+    initramfs-module-rundir \
+    initramfs-module-debug \
+    initramfs-module-createdisk \
+    lvm2 \
+"
+
+#VPF doesn't support addtional partition
+AOS_INITRAMFS_SCRIPTS:remove = "initramfs-module-opendisk"
```

Comment:
It looks like you don't need even more modules: initramfs-module-vardir, initramfs-module-machineid.
Better to specify under AOS_INITRAMFS_SCRIPTS only required modules instead of removing not required.

Reply:
Ok, I will correct it.